tools/wptrunner/wptrunner/wptcommandline.py

```diff
@@ -82,6 +83,24 @@ def create_parser(product_choices=None):
     mode_group.add_argument("--verify-log-full", action="store_true",
                             default=False,
                             help="Output per-iteration test results when running verify")
+    mode_group.add_argument("--verify-repeat-loop", action="store",
+                            default=10,
+                            help="Run tests in a loop for a number of times",
+                            type=int)
+    mode_group.add_argument("--verify-repeat-restart", action="store",
+                            default=5,
+                            help="Restart the whole process",
```

Comment:
"Number of iterations, for a run that restarts the runner between each iteration"